samples/operate-first/Makefile

```diff
@@ -2,9 +2,11 @@ ROOT_DIR := ../..
 include $(ROOT_DIR)/Makefile.env
 export RELEASE_NAME=fybrik
 export CRD_RELEASE_NAME=fybrik-crd
+export VAULT_RELEASE_NAME=fybrik-vault
```

Comment:
@eletonia In the quick-start the vault release name is called `vault` while the clusterRoleBinding that is generated in this pr refers to fybrik-vault service thus to avoid consistency issues I suggest VAULT_RELEASE_NAME would be also `vault`. 

`helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
    --set "vault.injector.enabled=false" \
    --set "vault.server.dev.enabled=true" \
    --values https://raw.githubusercontent.com/fybrik/fybrik/v0.4.0/charts/vault/env/dev/vault-single-cluster-values.yaml
kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s`

Reply:
Changed release name from `fybrik-vault` to `vault`